### ISSUE TITLE
champs suivi par mail

### ISSUE THREAD

**Issue by kptaine** [pluginsGLPI/formcreator#685]
Bonjour est-il possible de reprendre des champs des tickets GLPI pour les mettre dans un formulaire formcreator (ex : suivi par mail : oui/non ) ? 

Merci.

**Comment by btry** [comment]
Bonjour

D'une manière générale, il existe un certain nombre d'options dans la définition des tickets ou changements cibles. Les options se sont étoffées depuis la version 2.4.0, et il semble que le plugin couvre la plupart des besoins (connus). 

L'activation ou non des notifications se définit par le concepteur du formulaire, pas l'utilisateur. 

Notez dans la capture ci dessous qu'on peut éditer les acteurs de chaque type en choisissant si il faut envoyer les notifications.
![image](https://user-images.githubusercontent.com/14139801/28785263-23e75d0e-7616-11e7-8918-6b17da57fcee.png)

**Comment by kptaine** [comment]
pourtant quand on créé un ticket via glpi le demandeur à bien la possibilité de choisir s'il souhaite recevoir des notifications ou non 
![capture](https://user-images.githubusercontent.com/24512368/28819758-856885d2-76af-11e7-8c75-02e7644a1b17.PNG)

**Comment by btry** [comment]
Oui, mais le plugin a pour vocation de simplifier et encadrer / accompagner le demandeur dans sa démarche. 

Il est techniquement possible d'ajouter la fonctionnalité, et si c'st cela qu'il vous faut, je peux marquer ce ticket comme une demande de fonctionnalité. Il reste à définir comment le plugin gère l'activation des suivis quand il est fixé à la fois par l'utilisateur et par l'administrateur (et en contradiction).

**Comment by kptaine** [comment]
Je pense que c'est un fonctionnement qui peut satisfaire certains utilisateurs qui préfère suivre leurs incidents par l'interface pour certaine demande ou même ne recevoir aucune notification.
